server/test/services/migration/ProgramMigrationServiceTest.java

```diff
@@ -202,6 +207,37 @@ public void prepForExport_clearsNotificationPreferences() {
     assertThat(output.notificationPreferences()).isEmpty();
   }
 
+  @Test
+  public void prepForExport_clearsTiGroupAcls() {
```

Comment:
**thought:** Do we have bugs to allow for this or categories to be supported? If so, could be worth adding a TODO to these (and to the error code above).

Reply:
Great idea. I created [this issue](https://github.com/civiform/civiform/issues/8613) and added a TODO in as a comment.